### PR TITLE
Small copy fixes 

### PR DESCRIPTION
**Pull request by anikbrazeau**
- Change `Platform admin/Administrateur de la plateforme` to `Admin panel/Section admin` to be more concise and consistent
- Change `Your account settings` to `Your profile` to differentiate personal profile from services
- Change `All services` to `Your services` to be more precise
- Make tweaks to `Terms of use` page in Go Live process after sync with Lucas
- Remove confusing hint text on safelist page, not to confuse people with testing

### CODE REVIEW

**Comment by anikbrazeau** [comment]
[NAV Bug: Change label: "Platform admin" / "Administrateur de la plateforme"](https://trello.com/c/p8S5c5I1/348-nav-bug-change-label-platform-admin-administrateur-de-la-plateforme)

File: app/templates/views/platform-admin/reports.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% extends "views/platform-admin/_base_template.html" %}
 
 {% block per_page_title %}
-  Platform admin
+  Admin panel
```
Comment (AntoineAugusti) [inline app/templates/views/platform-admin/reports.html:4]:
```suggestion
  {{ _('Admin panel') }}
```
To translate the page title

File: app/templates/partials/nav/gc_header_nav.html [hunk 2]
```diff
@@ -88,16 +88,16 @@
...
+                {{ nav_menu_item(url_for('.api_integration', service_id=current_service.id),_('API integration'),'header--'+header_navigation.is_selected('api-integration')) }}
+                {{ nav_menu_item(url_for('.manage_users', service_id=current_service.id),_('Team members'),'header--'+header_navigation.is_selected('team-members')) }}
+                {{ nav_menu_item(url_for('.service_settings', service_id=current_service.id),_('Settings'),'header--'+header_navigation.is_selected('settings')) }}
                 {{ nav_menu_item(url_for('main.smtp_integration', service_id=current_service.id),_('SMTP'),'header--'+header_navigation.is_selected('smtp-integration')) }}
               {% else %} {# not current_user.has_permissions, i.e. services not in context #}
                 {{ nav_menu_item(url_for('main.choose_account'),_('Your services'),'pl-0 header--'+header_navigation.is_selected('choose_account'),id_key='choose_account') }}
-                {{ nav_menu_item(url_for('main.live_services', service_id=current_service.id),_('Platform admin')) }}
+                {{ nav_menu_item(url_for('main.live_services', service_id=current_service.id),_('Admin panel')) }}
```
Comment (AntoineAugusti) [inline app/templates/partials/nav/gc_header_nav.html:100]: Could you keep `main.` in URLs?

**Review by AntoineAugusti — CHANGES_REQUESTED** [review]
Thanks for submitting these content changes yourself!

**Review by AntoineAugusti — APPROVED** [review]
Wonderful 🚢

**Pull request merged**